tools/c7n_left/c7n_left/output.py

```diff
@@ -83,19 +83,31 @@ def __init__(self, ctx, config):
         super().__init__(ctx, config)
         self.console = Console(file=config.output_file)
         self.started = None
+        self.matches = 0
 
-    def on_execution_started(self, policies):
-        self.console.print("Running %d policies" % (len(policies),))
+    def on_execution_started(self, policies, graph):
+        self.console.print(
+            "Running %d policies on %d resources" % (len(policies), len(graph))
+        )
         self.started = time.time()
 
     def on_execution_ended(self):
         self.console.print(
-            "Execution complete %0.2f seconds" % (time.time() - self.started)
+            "Evaluation complete %0.2f seconds -> %s"
+            % (
+                time.time() - self.started,
+                (
+                    not self.matches
```

Comment:
I find this weird to read the way its formatted.   Could we do something not-inline to make it more obvious?

```
message = "[green]Success[green]"
if matches:
    message = "[red]%d Failures[/red]" % len(self.matches)
```

Reply:
sounds good.